syft/generic/pointers/pointer_tensor.py

```diff
@@ -175,77 +176,6 @@ def get_class_attributes(self):
             "garbage_collect_data": self.garbage_collect_data,
         }
 
-    @staticmethod
```

Comment:
Note that this `create_pointer` method is actually for creating pointers to plain tensors (not for creating pointers to `PointerTensors`.) It was only used in one place in PySyft, so it has been combined into the `TorchTensor.create_pointer()` method.